rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -890,10 +890,12 @@ extern __m128i sse_cellbe_lvrx_v0(u64 addr);
 extern void sse_cellbe_stvlx_v0(u64 addr, __m128i a);
 extern void sse_cellbe_stvrx_v0(u64 addr, __m128i a);
 
-[[noreturn]] static void ppu_trap(ppu_thread& ppu, u64 addr)
+static void ppu_trap(ppu_thread& ppu, u64 addr)
```

Comment:
Unfortunately, I cannot merge it. Returning from trap is incompatible with current LLVM recompiler, and fixing it is impossible without cache version bump which I would like to avoid.

Reply:
I see, that's unfortunate. Maybe you can keep it in mind for the next time you are required to make a version bump?

Reply:
Alternatively, would removing the change to the recompiler trap implementation be good enough for a merge?